### ISSUE TITLE
Cannot kill the last werewolf

### ISSUE THREAD

**Issue by hjwylde** [hjwylde/werewolf#26]
```
bash-3.2$ werewolf --caller thong.kuah vote eilis
werewolf: Prelude.last: empty list
```

```
Game {_turn = Villagers, _players = [Player {_name = "lucy", _role = Role {_name = "Villager", _allegiance = Villagers, _description = "An ordinary townsperson humbly living in Millers Hollow.", _advice = "Bluffing can be a good technique, but you had better be convincing about what you say."}, _state = Dead},Player {_name = "tiger", _role = Role {_name = "Werewolf", _allegiance = Werewolves, _description = "A shapeshifting townsperson that, at night, hunts the residents of Millers Hollow.", _advice = "Voting against your partner can be a good way to deflect suspicion from yourself."}, _state = Dead},Player {_name = "hjwylde", _role = Role {_name = "Villager", _allegiance = Villagers, _description = "An ordinary townsperson humbly living in Millers Hollow.", _advice = "Bluffing can be a good technique, but you had better be convincing about what you say."}, _state = Dead},Player {_name = "thong.kuah", _role = Role {_name = "Villager", _allegiance = Villagers, _description = "An ordinary townsperson humbly living in Millers Hollow.", _advice = "Bluffing can be a good technique, but you had better be convincing about what you say."}, _state = Alive},Player {_name = "michael.fowler", _role = Role {_name = "Villager", _allegiance = Villagers, _description = "An ordinary townsperson humbly living in Millers Hollow.", _advice = "Bluffing can be a good technique, but you had better be convincing about what you say."}, _state = Alive},Player {_name = "eilis", _role = Role {_name = "Werewolf", _allegiance = Werewolves, _description = "A shapeshifting townsperson that, at night, hunts the residents of Millers Hollow.", _advice = "Voting against your partner can be a good way to deflect suspicion from yourself."}, _state = Alive},Player {_name = "aj", _role = Role {_name = "Villager", _allegiance = Villagers, _description = "An ordinary townsperson humbly living in Millers Hollow.", _advice = "Bluffing can be a good technique, but you had better be convincing about what you say."}, _state = Dead}], _sees = fromList [], _votes = fromList [("eilis","thong.kuah"),("michael.fowler","eilis")]}
```

**Comment by hjwylde** [comment]
Since v0.2.0.0.

**Comment by hjwylde** [comment]
Fixed in v0.2.0.1, v0.3.0.3 and v0.3.1.1.
